src/main/java/org/mockito/plugins/InlineMockMaker.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2019 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+
+package org.mockito.plugins;
+
+import org.mockito.Incubating;
+
+/**
+ * Extension to {@link MockMaker} for mock makers that changes inline method implementations.
+ * @since 2.24.8
+ */
+@Incubating
+public interface InlineMockMaker extends MockMaker {
+    /**
+     * Cleans up internal state for specified {@code mock}. You may assume there won't be any interaction to this {@code mock}
+     * after this is called.
```

Comment:
Lets add an additional sentence that this method can be called with null, indicating all mocks should be cleared

Reply:
Done.